src/Either.js

```diff
@@ -127,4 +127,12 @@ Either.Left = function(value) {
 };
 
 
+// Fold
+Either.prototype.fold = function eitherFold(leftFn, rightFn) {
+  if (this instanceof _Left) {
+    return leftFn(this.value);
+  }
+  return rightFn(this.value);
+};
```

Comment:
rather than relying on `instanceof`, let's use the `isLeft` property (which sanctuary also uses), and write this as a ternary expression, e.g.

`return (this.isLeft) ? leftFn(this.value) : rightFn(this.value);`